addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -894,6 +894,54 @@ export function preserveCursor(document) {
     };
 }
 
+export function getParentBehaviorElement(node) {
+    if (!node) {
+        return null;
+    }
+    if (node && node.classList.contains("o_field_html")) {
```

Comment:
The ``... node && ...`` condition is not necessary here, because you check just above that node was not falsy.

I think we could rewrite the two return guard like so : 

```javascript
if (!node || node.classList.contains("o_field_html")) {
    return null;
}
```